docs/harvesting.md

```diff
@@ -0,0 +1,222 @@
+# Harvesting
+
+Harvesting is the process of fetching of automatically remote metadata (ie. from other data portals or not)
+and store them into udata for being able to search them.
```

Comment:
Wow, so many of typos in my sentence :scream: 
I'll fix that ASAP. But I think, we need to keep ``fetching` as `syncing` suggest it's bidirectionnal while harvesting is not.